src/components/touchableOpacity/index.tsx

```diff
@@ -45,7 +43,9 @@ export type TouchableOpacityProps = Omit<RNTouchableOpacityProps, 'style' | 'onP
     customValue?: any;
     style?: StyleProp<ViewStyle> | Animated.AnimatedProps<StyleProp<ViewStyle>>;
     onPress?: (props: TouchableOpacityProps) => void;
-  };
+}
+
+export type TouchableOpacityPropTypes = TouchableOpacityProps; //TODO: remove after ComponentPropTypes deprecation;
```

Comment:
@vikasg603 
One last comment, what is that for? (:

Reply:
It was there in your View component.
I just copied the same code with some small changes :)

Reply:
It's not required here (: